### PR TITLE
Add spconv python package rosdep keys

### PR DESCRIPTION
**Pull request by JWhitleyWork**
Please add the following dependency to the rosdep database.

## Package name:

`spconv`: Spatially Sparce Convolutional Library

## Package Upstream Source:

https://github.com/traveller59/spconv

## Purpose of using this:

This is a dependency of several SLAM algorithms including hdl_graph_slam and hdl_localization. These provide some of the best results that I've seen for map generation and localization.

## Links to Distribution Packages
- https://pypi.org/project/spconv/
- https://pypi.org/project/spconv-cu114/
- https://pypi.org/project/spconv-cu116/
- https://pypi.org/project/spconv-cu117/
- https://pypi.org/project/spconv-cu118/

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -9209,6 +9209,56 @@ python3-sparkfun-ublox-gps-pip:
...
+  debian:
+    pip:
+      packages: [spconv]
+  fedora:
+    pip:
+      packages: [spconv]
+  ubuntu:
+    pip:
+      packages: [spconv]
+python3-spconv-cuda-114-pip:
```
Comment (ivanpauno) [inline rosdep/python.yaml:9222]: could you use the pip name?
i.e. `python3-spconv-cu114-pip`

**Review by ivanpauno — CHANGES_REQUESTED** [review]

**Comment by nuclearsandwich** [comment]
@JWhitleyWork friendly ping on this rosdep change.

File: rosdep/python.yaml (hunk 1 above)
Comment (JWhitleyWork) [reply rosdep/python.yaml]: Sorry for the delay. Applied!

**Comment by JWhitleyWork** [comment]
@nuclearsandwich Done!

**Review by ivanpauno — APPROVED** [review]

**Review by nuclearsandwich — APPROVED** [review]

File: rosdep/python.yaml [hunk 2]
```diff
@@ -9321,6 +9321,56 @@ python3-sparkfun-ublox-gps-pip:
...
+python3-spconv-pip:
+  debian:
+    pip:
+      packages: [spconv]
+  fedora:
+    pip:
+      packages: [spconv]
+  ubuntu:
+    pip:
+      packages: [spconv]
```
Comment (nuclearsandwich) [inline rosdep/python.yaml:9333]: @JWhitleyWork CI is catching that this key should come after the others due to lexicographical sorting.

**Review by nuclearsandwich — CHANGES_REQUESTED** [review]

**Comment by clalancette** [comment]
No response to this in over a month, so closing.  @JWhitleyWork feel free to reopen and respond to the comments.

**Comment by JWhitleyWork** [comment]
@clalancette / @nuclearsandwich Sorry for the delayed response. Please re-open and re-evalute this PR. It should now pass nosetests.

**Comment by tfoote** [comment]
@JWhitleyWork Unfortunately GitHub won't let us reopen it now that you've either force pushed or recreated the branch. You'll have to create a new PR. Please just refer back to this in the new one.

**Pull request closed without merging**